src/borg/archiver.py

```diff
@@ -2866,6 +2867,17 @@ def define_archive_filters_group(subparser, *, sort_by=True, first_last=True):
         - The archive checks can be time consuming, they can be skipped using the
           ``--repository-only`` option.
 
+        The ``--max-duration`` option can be used to split a long-running repository check into multiple partial checks.
+        After the given number of seconds the check is interrupted. The next partial check will continue where the
+        previous one stopped, until 100% of the repository have been checked. Example: Running a check with
```

Comment:
... until the complete repository has been checked.

(avoid % in case we use this as a format string)